style_rwd.css

```diff
@@ -1,22 +1,67 @@
 @media (max-width: 767px){
     img.main_photo {
         display: none;
+        width:auto;
     }
-    .catch_phrase{
-        max-width: 767px;
+    .section_hero {
+        padding-bottom: 42px;
+    }
+    body{
+        width: auto;
     }
     .column{
+        width: auto;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
+        font-size: 40px;
         font-weight: 30;
-        max-width: 767px;
+        margin-bottom: 0;
+        width: auto;
+    }
+    .button_container {
+        justify-content: center;
+        width: auto;
+        margin: 0;
     }
-    .btn{
+    .btn {
+        border-radius: 32px;
```

Comment:
don’t set radius in px, use relative values like %

Reply:
ye we've been here but as before, there is no % value which I can keep the curvature :)